src/client/Client.cc

```diff
@@ -2402,6 +2408,7 @@ void Client::handle_client_session(const MConstRef<MClientSession>& m)
       if (session->state == MetaSession::STATE_OPEN) {
         ldout(cct, 10) << "mds." << from << " already opened, ignore it"
                        << dendl;
+	reinit_mds_features(session.get(), m);
```

Comment:
similar to my comment https://tracker.ceph.com/issues/63188#note-6 by is it possible to first check the feature bits held by the client vs what the mds holds currently? if not similar then reinit else dont.

Reply:
Even if the features match, it doesn't hurt to just refresh those, isn't it? (compare+assign vs assign)

Reply:
okay so there are no drawbacks involved